superset/charts/schemas.py

```diff
@@ -705,7 +705,94 @@ class ChartDataExtrasSchema(Schema):
     )
 
 
+class AnnotationLayerSchema(Schema):
+    annotationType = fields.String(
```

Comment:
We are mixing camel case with snake case, can we change it?

Reply:
Unfortunately not, this is what the state from the control looks like, and it would feel wrong to map it over from camel to kebab.